contxt/services/facilities.py

```diff
@@ -1,27 +1,27 @@
-from typing import Optional
+from typing import List, Optional
 
 from contxt.auth import Auth
 from contxt.models.facilities import Facility
-from contxt.services.api import ConfiguredApiService
+from contxt.services.api import ConfiguredApi
 from contxt.services.assets import AssetsService
 from contxt.utils import make_logger
 
 logger = make_logger(__name__)
 
 
-class FacilitiesService(ConfiguredApiService):
+class FacilitiesService(ConfiguredApi):
     """
     Service to interact with our Facilities API.
 
     NOTE: The facility_id in this service is the legacy integer id.
     """
 
-    _configs = AssetsService._configs
+    _envs = AssetsService._envs
 
     def __init__(self, auth: Auth, env: str = "production"):
-        super().__init__(auth, env)
+        super().__init__(env, auth)
 
```

Comment:
Do any of the below get facilities calls map directly back to a CLI command that needs the pretty print?

Reply:
that's likely, i'm going to tackle cli-related updates in a separate PR